data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
```

Comment:
Please add table attributes before inserting their values as this ` INSERT INTO animals (animal_name, date_of_birth, escape_attempts, neutered, weight_kg)`.

Reply:
I did it, but just a question, is this a good practice?